emu/src/cpu/arm7tdmi.rs

```diff
@@ -611,9 +611,13 @@ impl Arm7tdmi {
 
     pub fn pc_relative_load(&mut self, op_code: ThumbModeOpcode) -> Option<u32> {
         let rd = op_code.get_bits(8..=10);
-        let value = op_code.get_bits(0..=7);
+        let address = op_code.get_bits(0..=7) as usize;
+        let mut pc = self.registers.program_counter() as u32;
+        pc.set_bit_off(0);
```

Comment:
```suggestion
        pc.set_bit_off(1);
```

Last one, sorry! In THUMB we're sure we've bit 0 to 0 since we set it before fetching, we have to set bit 1 to 0 here.

Reply:
Oh yes because only in Arm mode we set both to zero, right